src/foam/mlang/mlang.js

```diff
@@ -4350,6 +4350,58 @@ foam.CLASS({
   ]
 });
 
+foam.CLASS({
+  package: 'foam.mlang',
+  name: 'If',
+  extends: 'foam.mlang.AbstractExpr',
+
+  properties: [
+    {
+      class: 'foam.mlang.predicate.PredicateProperty',
+      name: 'predicate',
+      javaFactory: 'return foam.mlang.MLang.TRUE;'
+    },
+    {
+      class: 'foam.mlang.ExprProperty',
+      name: 'valueIfTrue'
```

Comment:
maybe better names: trueExpr and falseExpr

Reply:
I've renamed these to trueExpr, falseExpr in my payblii branch.